app/js/views/reminders/reminder-item.jsx

```diff
@@ -1,12 +1,19 @@
+/* global TwitterCldr, TwitterCldrDataBundle */
+
 import React from 'components/react';
 import moment from 'components/moment';
+import 'components/cldr/en';
+import 'components/cldr/core';
 
 const COLOURS = ['red', 'orange', 'green', 'blue', 'violet'];
 
 export default class ReminderItem extends React.Component {
   constructor(props) {
     super(props);
 
+    TwitterCldr.set_data(TwitterCldrDataBundle);
```

Comment:
It's not cool that we have to do this :(.  If we were using the same sort of build system as we had in `JsSpeechRecognizer` we could just use it as an ES6/node module (as described in the `twitter-cldr-npm` module's README.  I think using `rollup` + `rollup-plugin-commonjs` + `rollup-plugin-node-resolve` is a better workflow.  We then wouldn't need to list out files to copy in the gulpfile either as rollup would resolve them from `node_modules` automatically.  If you agree we can do this in a separate PR.